ui/src/pages/AllStories/index.tsx

```diff
@@ -438,6 +591,18 @@ export const AllStories: React.FC = () => {
                 handleRequestSort("author_country");
               },
             },
+            {
+              name: "tags",
+              header: "Tags",
+              width: 300,
+              onHeaderClick() {
+                handleRequestSort("tags");
+              },
+              cell: (story) =>
+                story.tags.map((tag) => (
+                  <StyledChip color="primary" key={story.id} label={tag} />
```

Comment:
```suggestion
                  <StyledChip color="primary" key={story.ID} label={tag} />
```

Reply:
I dont think this should be story.ID because then stories with multiple tags would have duplicate key and React throws an error msg for that

Reply:
Why can't we keep it as `key={tag}` so you know it's unique? we shouldn't be using `story.id` anywhere because that's part of the table cell id and it's confusing with` story.ID`

Reply:
yeah kept it as key={tag}